packages/python/javdb_integrations/rclone_manager.py

```
@@ -1360,19 +1413,84 @@ def on_rows(rows: list):
+                )
+                if _created_local_staging_session:
+                    db_mark_session_committed(_staging_session_id)
+            except Exception as e:
+                logger.error(
+                    f"Failed to swap RcloneInventory staging — "
+                    f"main table left UNCHANGED: {e}"
+                )
+                try:
+                    db_drop_rclone_staging(_staging_session_id)
+                except Exception:
+                    pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)